tests/platform_tests/link_flap/test_cont_link_flap.py

```diff
@@ -28,6 +28,13 @@ class TestContLinkFlap(object):
     TestContLinkFlap class for continuous link flap
     """
 
+    def get_bgp_memory_usage(self, duthost):
+        bgp_memory_output = duthost.shell('vtysh -c "show memory bgp"')["stdout"]
```

Comment:
Better to check more memory usage in FRR, like zebra

Reply:
Done, added zebra.